blazor/dialog/predefined-dialogs.md

```diff
@@ -0,0 +1,338 @@
+{% include_relative code-snippet/confirm-dialog.razor %}
+
+{% endhighlight %}
+{% endtabs %}
+
+![Confirm Dialog](./images/blazor-confirm-dialog.png)
+
+### Prompt
+
+A [Prompt](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.SfDialogService.html#Syncfusion_Blazor_Popups_SfDialogService_PromptAsync_System_String_System_String_Syncfusion_Blazor_Popups_DialogOptions_) dialog is used to get input from the user. When the user enters "OK", the input value is returned. When they click "Cancel", the null value  returned.
+
+Use the following code to render a simple prompt dialog in an application.
+
+{% tabs %}
+{% highlight razor %}
+
+{% include_relative code-snippet/prompt-dialog.razor %}
+
+{% endhighlight %}
+{% endtabs %}
+
+![Prompt Dialog](./images/blazor-prompt-dialog.png)
+
+## Dragging
+
+The Dialog supports to drag within its target container by grabbing the Dialog header, which allows the user to reposition the Dialog dynamically by using [AllowDragging](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.SfDialog.html#Syncfusion_Blazor_Popups_SfDialog_AllowDragging) property.
+
+{% tabs %}
+
+{% highlight tabtitle="alert.razor" %}
+{% include_relative code-snippet/alert-drag.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="confirm.razor" %}
+{% include_relative code-snippet/confirm-drag.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="prompt.razor" %}
+{% include_relative code-snippet/prompt-drag.razor %}
+{% endhighlight %}
+
+{% endtabs %}
+
+## Animations
+
+The predefined ialog can be animated during the open and close actions. Also, users can customize animation’s [DialogAnimationSettings.Delay](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.DialogAnimationSettings.html#Syncfusion_Blazor_Popups_DialogAnimationSettings_Delay), [DialogAnimationSettings.Duration](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.DialogAnimationSettings.html#Syncfusion_Blazor_Popups_DialogAnimationSettings_Duration) and [DialogAnimationSettings.Effect](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.DialogAnimationSettings.html#Syncfusion_Blazor_Popups_DialogAnimationSettings_Effect) by using the `DialogAnimationSettings` property.
+
+In the following sample, `Zoom` effect is enabled. So, The Dialog will open with `ZoomIn` and close with `ZoomOut` effects.
+
+{% tabs %}
+
+{% highlight tabtitle="alert.razor" %}
+{% include_relative code-snippet/alert-animation.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="confirm.razor" %}
+{% include_relative code-snippet/confirm-animation.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="prompt.razor" %}
+{% include_relative code-snippet/prompt-animation.razor %}
+{% endhighlight %}
+
+{% endtabs %}
+
+## Position
+
+You can customize the dialog position by using [Position](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.Position.html#fields) property. Use the following code to customize the dialog position.
```

Comment:
You can customize the dialog position by using the [Position](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.Position.html#fields) property. Use the following code to customize the dialog position: